apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -1981,7 +1981,7 @@ else:
           onComplete: function (id, fileName, response) {
             self.pendingUploads(self.pendingUploads() - 1);
             if (response.status != 0) {
-              $(document).trigger('error', "${ _('Error: ') }" + response.data);
+              $(document).trigger('error', "${ _('Upload Error: No write permission') }");
```

Comment:
Is it always a write permission issue for status != 0?

Reply:
Very probably not!